Assets/Scripts/Gamemode/GamemodeBase.cs

```diff
@@ -204,6 +204,7 @@ public virtual void OnTitanKilled(string titanName)
             }
         }
 
+        //would be better to get float value for time and totalRoomTime instead as those are float value to begin with so this method won't have any loss of infos
```

Comment:
Time should be managed via TimeService, but why place a comment here?

Reply:
I'll check this timeservice thingh

Reply:
TimeService is currently only available in Issue 160 however: https://github.com/AoTTG-2/AoTTG-2/blob/%23160-titan-difficulty/Assets/Scripts/Services/TimeService.cs

Will try to do the partial merge asap as discussed so you can use that one.